### PR TITLE
feat(Scripts/Spells): Move to spell scripts Deathbolt

### PR DESCRIPTION
**Pull request by IntelligentQuantum**
## Changes Proposed:
-  
-  

## Issues Addressed:

- Closes 

## Tests Performed:

- 
- 

## How to Test the Changes:

1.
2.
3.

## Known Issues and TODO List:

- [ ]
- [ ]

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

**Comment by Lynethris** [comment]
Tested with .cast 51854 and with Thalgran Blightbringer, no issue find.

File: src/server/scripts/Northrend/zone_sholazar_basin.cpp [hunk 1]
```diff
@@ -1505,6 +1492,34 @@ class spell_shango_tracks : public SpellScriptLoader
...
+
+    void HandleScriptEffect(SpellEffIndex /* effIndex */)
+    {
+        Unit* caster = GetCaster();
+        Unit* target = GetHitUnit();
+
+        if (target->HasAura(SPELL_FREYAS_WARD))
+            target->CastSpell(caster, SPELL_SEVENFOLD_RETRIBUTION, true);
+        else
+            caster->CastSpell(target, SPELL_DEATHBOLT, true);
```
Comment (Winfidonarleyan) [inline src/server/scripts/Northrend/zone_sholazar_basin.cpp:1514]: brackets

**Review by Winfidonarleyan — APPROVED** [review]

File: src/server/scripts/Northrend/zone_sholazar_basin.cpp [hunk 2]
```diff
@@ -1505,6 +1492,34 @@ class spell_shango_tracks : public SpellScriptLoader
...
+};
+
+class spell_q12611_deathbolt : public SpellScript
+{
+    PrepareSpellScript(spell_q12611_deathbolt);
+
+    void HandleScriptEffect(SpellEffIndex /* effIndex */)
+    {
+        Unit* caster = GetCaster();
+        Unit* target = GetHitUnit();
```
Comment (Nefertumm) [inline src/server/scripts/Northrend/zone_sholazar_basin.cpp:1509]: No checks needed?

Comment (IntelligentQuantum) [reply src/server/scripts/Northrend/zone_sholazar_basin.cpp:1509]: No, but i can add if you want

**Pull request merged**